cmd/cli/kubectl-kyverno/commands/apply/command_test.go

```diff
@@ -823,3 +839,49 @@ func TestCommandHelp(t *testing.T) {
 	assert.NoError(t, err)
 	assert.True(t, strings.HasPrefix(string(out), cmd.Long))
 }
+func Test_Apply_MutatingAdmissionPolicies(t *testing.T) {
```

Comment:
There are three tests that don't exist here:
1. test/cli/test-mutating-admission-policy/match-condition
2. test/cli/test-mutating-admission-policy/check-ns-add-label
3. test/cli/test-mutating-admission-policy/check-labels

Can we add the above tests?